perma_web/static/js/links-list.js

```diff
@@ -158,6 +168,56 @@ $(function() {
         }, 0);
     }
 
+    function findNodeBySavedFolder () {
+      var folder = localStorage.getItem("perma_selected_folder"),
+        parsedFolder = JSON.parse(folder),
+        node;
+
+      var searchAttrs = "";
+
+      if (parsedFolder.orgID) {
+        searchAttrs += "[data-organization_id="+parsedFolder['orgID']+"]"
+      }
+
+      if (parsedFolder.folderID) {
+        searchAttrs += "[data-folder_id="+parsedFolder['folderID']+"]"
+      }
+
+      node = folderTree.get_node(searchAttrs);
+      // if no node exists,
+      // we're looking at "My Links", not an organization
+      if (!node) {
+        node = folderTree.get_node('ul > li:first');
+      }
+
+      folderTree.select_node(node);
+
+      return node;
+    }
+    function getFolderByNode(node) {
+      var orgID  = node.data.organization_id,
+        folderID = node.data.folder_id,
+        folder   = {'orgID':orgID,'folderID':folderID};
+      return folder;
+    }
+
+    function updateLocalStorage(node) {
+      var folder = getFolderByNode(node)
+      localStorage.setItem("perma_selected_node", JSON.stringify(node));
+      localStorage.setItem("perma_selected_folder", JSON.stringify(folder));
+    }
+
+    function updatePathWithSelected(node) {
+      var nodePath = folderTree.get_path(node).join(" &gt; ");
+      var myLinks = nodePath.includes("My Links");
```

Comment:
I'm assuming that there are no other folders called My Links. Faulty assumption, but might work in the meantime?